mobile_rewards/src/server.rs

```diff
@@ -267,48 +247,98 @@ impl Server {
         Ok(())
     }
 
-    async fn handle_rewards(
-        &mut self,
-        reward_period: Range<u64>,
```

Comment:
this reward period is still needed to correctly set the start and end _block height_ in the subnetwork reward transaction.